src/pcapi/core/offers/api.py

```diff
@@ -48,3 +56,24 @@ def list_offers_for_pro_user(
         name_keywords=name_keywords,
         requested_status=requested_status,
     )
+
+
+def create_offer(offer_data: PostOfferBodyModel, user: UserSQLEntity) -> OfferSQLEntity:
+    venue = load_or_raise_error(VenueSQLEntity, offer_data.venue_id)
+
+    ensure_current_user_has_rights(RightsType.editor, venue.managingOffererId)
```

Comment:
```suggestion
    ensure_current_user_has_rights(RightsType.editor, venue.managingOffererId, user=user)
```

Sinon `ensure_current_user_has_rights()` utilise le fameux `current_user` global. Pour l'instant c'est le même, mais plus tard... (D'ailleurs je trouve ce genre de valeur par défaut un peu dangereux.)

Reply:
:skull_and_crossbones: en effet